deploy/helm/sumologic/templates/hpa.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if and .Values.sumologic.fluentd.autoscaling.enabled}}
```

Comment:
what's the reason for `and` here? We're not checking two conditions.

Reply:
Good catch! Will fix it.